pkg/cmd/cli/cmd/project.go

```diff
@@ -115,8 +130,68 @@ func NewCmdProject(f *clientcmd.Factory, out io.Writer) *cobra.Command {
 				glog.Fatalf("Error saving project information in the config: %v.", err)
 			}
 
-			fmt.Printf("Now using project '%v'.\n", project.Name)
+			fmt.Printf("Now using context '%v' with project '%v'.\n", contextInUse, namespaceInUse)
 		},
 	}
 	return cmd
 }
+
+func getProject(oClient *client.Client, name string) (*api.Project, error) {
+	project, err := oClient.Projects().Get(name)
+
+	if err != nil {
+		if errors.IsNotFound(err) || clientcmd.IsForbidden(err) {
+			msg := fmt.Sprintf("A project named '%v' does not exist or you do not have rights to view it.", name)
+
+			if projectsNames, err := getProjectsNames(oClient); err != nil {
+				msg += fmt.Sprintf(" Your projects are: %v.", strings.Join(projectsNames, ", "))
```

Comment:
Also, "on this server".  Then, at the end of the message, if they have a different server in their config, display a message: "To see projects on another server, pass '--server='"